test/plots/piePlotTests.ts

```diff
@@ -13,6 +13,51 @@ describe("Plots", () => {
       svg.remove();
     });
 
+    it("each slice consists of a fill path and a stroke path", () => {
+      let svg = TestMethods.generateSVG(500, 500);
+      let piePlot = new Plottable.Plots.Pie();
+      piePlot.sectorValue((d) => d.value);
+      let data = [
+        { value: 1 },
+        { value: 1 }
+      ];
+      let dataset = new Plottable.Dataset(data);
+      piePlot.addDataset(dataset);
+      piePlot.renderTo(svg);
+      let renderArea = (<any> piePlot)._renderArea;
+      let arcPaths = renderArea.selectAll(".arc");
+      let arcFillPaths = renderArea.selectAll(".arc.fill");
+      let arcOutlinePaths = renderArea.selectAll(".arc.outline");
+      assert.strictEqual(arcPaths.size(), 4, "2 paths per datum");
+      assert.strictEqual(arcFillPaths.size(), 2, "1 fill path per datum");
+      assert.strictEqual(arcOutlinePaths.size(), 2, "1 outline path per datum");
+      assert.strictEqual(arcFillPaths.style("stroke"), "none", "fill paths have no stroke");
+      assert.strictEqual(arcOutlinePaths.style("fill"), "none", "outline paths have no fill");
+      svg.remove();
+    });
+
+    it("each entity selection consists of a fill path and a stroke path", () => {
+      let svg = TestMethods.generateSVG(500, 500);
+      let piePlot = new Plottable.Plots.Pie();
+      piePlot.sectorValue((d) => d.value);
+      let data = [
+        { value: 1 },
+        { value: 1 }
+      ];
+      let dataset = new Plottable.Dataset(data);
+      piePlot.addDataset(dataset);
+      piePlot.renderTo(svg);
+
+      let entities = piePlot.entities();
+      entities.forEach((entity) => {
+        assert.strictEqual(entity.selection.size(), 2, "each entity selection has 2 paths");
+        assert.lengthOf(entity.selection.filter("fill"), 1, "each entity selection has 1 fill path");
+        assert.lengthOf(entity.selection.filter("outline"), 1, "each entity selection has 1 stroke path");
```

Comment:
Shouldn't the filters here be `".fill"` and `".outline"`?

D3 selections always have size 1. The correct way check their size is this:

```Typescript
assert.strictEquals(selection.size(), expectedSize, message);
```